components/x-gift-article/src/HighlightSavedMessage.jsx

```diff
@@ -0,0 +1,25 @@
+import { h } from '@financial-times/x-engine'
+
+export default ({ handleCloseSuccessMessage }) => {
+	return (
+		<div
+			className="o-message o-message--alert o-message--success x-gift-article__highlights-saved-message"
+			data-o-component="o-message"
+		>
+			<div className="o-message__container">
+				<div className="o-message__content">
+					<p className="o-message__content-main">
+						<span className="o-message__content-highlight">Highlights saved.</span>
+					</p>
+					<button
+						type="button"
+						className="o-message__close"
+						aria-label="close"
```

Comment:
**Suggestion:** Instead of relying on `aria-label` here it might be nicer to add text to the content but visually hide that text. That will provide more reliable results for assistive tech